### PR TITLE
Fix build on Windows

### PR DESCRIPTION
**Pull request by neykov**
Build fails, trying to resolve the http url as a file system path.

### CODE REVIEW

File: karaf/features/src/main/feature/feature.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
     <repository>mvn:org.apache.karaf.features/spring/${karaf.version}/xml/features</repository>
 
     <!-- TODO how to add central.maven.org repo? -->
```
Comment (bostko) [inline karaf/features/src/main/feature/feature.xml]: Is this comment still needed?

Comment (neykov) [inline karaf/features/src/main/feature/feature.xml]: Thanks for pointing it out, removed.

**Comment by geomacy** [comment]
👍 looks good to me

**Comment by neykov** [comment]
Thanks @bostko, @geomacy. Merging.

**Pull request merged**